app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    @classmethod
+    def __repr__(cls) -> str:
+        return f"{cls.people}"
+
+
+def create_person_list(people: dict) -> list:
```

Comment:
`people` argument expects not dict, but a list of dicts. Besides that, you could specify that you return a list of Person instances in return type annotation.